tests/web/WebDriverTest.php

```diff
@@ -413,14 +413,21 @@ public function testWebDriverByLocators()
         $this->module->seeElement(WebDriverBy::cssSelector('form.global_form_box'));
         $this->module->seeElement(WebDriverBy::xpath(\Codeception\Util\Locator::tabIndex(4)));
         $this->module->fillField(WebDriverBy::name('password'), '123456');
+        
         $this->module->amOnPage('/form/select');
         $this->module->selectOption(WebDriverBy::name('age'), 'child');
+        
         $this->module->amOnPage('/form/checkbox');
         $this->module->checkOption(WebDriverBy::name('terms'));
+        
         $this->module->amOnPage('/');
         $this->module->seeElement(WebDriverBy::linkText('Test'));
         $this->module->click(WebDriverBy::linkText('Test'));
         $this->module->seeCurrentUrlEquals('/form/hidden');
+
+        $this->module->amOnPage('/form/example7');
+        $this->module->click('Chocolate Bar');
```

Comment:
A problem with this test is that it doesn't test your change.
It passes with PhpBrowser and unpatched WebDriver module, because `Chocolate Bar` matches
`<span>Buy Chocolate Bar</span>`
https://github.com/Codeception/Codeception/blob/2.2/tests/data/app/view/form/example7.php